### PR TITLE
Addtravis

### PR DESCRIPTION
**Pull request by danischroeter**
This would add travisci support (obviously with your help as the admin of the project...)
Thx!

### CODE REVIEW

**Comment by ivantopo** [comment]
Hey @danischroeter, could you please update this PR to have just the travis file and the build badge on the readme? I thought the diff would be smaller after merging #14 but it isn't :confused:

**Comment by danischroeter** [comment]
> Hey @danischroeter, could you please update this PR to have just the travis file and the build badge on the readme? I thought the diff would be smaller after merging #14 but it isn't 😕

Sure thing - was based on the 2.0 migration but with an older version...
its rebased now

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: scala
+script:
+  - sbt test
```
Comment (ivantopo) [inline .travis.yml]: Please, make it `+test` so it goes through all supported Scala versions.

**Review by ivantopo — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (danischroeter) [reply .travis.yml]: sure - also fixed issues when running with +test

**Pull request merged**